packages/launchpad/cypress/e2e/config-warning.cy.ts

```diff
@@ -209,7 +209,8 @@ describe('component testing dependency warnings', () => {
     cy.get('.warning-markdown').find('li').should('have.length', 3)
   })
 
-  it('warns against outdated @vue/cli dependency', () => {
+  // TODO: make this version of @vue/cli-service 5 in https://github.com/cypress-io/cypress/issues/30295 or related issue
```

Comment:
this test will probably get deleted since we are removing `@vue/cli-service` support entirely, but we need to support vue with webpack. scaffolding tbd